### ISSUE TITLE
Cannot roll initiative from combat chat

### ISSUE THREAD

**Issue by LEronsn** [Oromis/touch-vtt#88]
Cannot roll initiative from combat chat. Clicking the dice buttons do nothing. You can rearrange the combatants though.
Maybe something similar like in the scene chat will do the trick.
Thanks in advance

**Comment by Aioros** [comment]
Hi, I can't seem to reproduce this, or maybe I'm misunderstanding. Are we talking about this tab?

![Animation](https://github.com/user-attachments/assets/e10c7d2d-c1c1-478f-820f-55f0063cb463)

I'm also not sure what you mean by rearrange the combatants.
In any case, let me know what Foundry version you're using. And if you could get a little screen/video of the issue that would be super helpful.

**Comment by LEronsn** [comment]
Yes, that's the tab I was talking about. For me it's not working to click the dice button. How do you record your screen?

**Comment by Aioros** [comment]
I used ScreenToGif to record, but there's probably many other tools if you look for something like "record screen to gif" or similar. Just to make sure, when you say "a fresh pathfinder system", do you mean without any other modules?

**Comment by Aioros** [comment]
Actually, don't worry about the recording. I am able to replicate it now, it's a specific pf2e thing. I'll keep you posted.

**Comment by Aioros** [comment]
I believe this should now be fixed in v2.2.5. Let me know if it works for you after updating.

**Comment by LEronsn** [comment]
That was fast. Yep, Pathfinder 2e combat works great now. Thanks again 👍😊
